classes.py

```diff
@@ -20,6 +21,31 @@ def __init__(self):
     self.enemy_size = 50
 
     self.alive_enemies = []
+    self.state_of_game = False # False = waiting for user to start game, True = shooting down enemies
+
+    try:
+      with open("highscore.json", "r") as file:
+        highscore = json.load(file)
+        self.highscore = highscore["highscore"]
+    except FileNotFoundError:
+      self.highscore = 0
+
+    self.score_text = self.font.render("Score:", True, self.text_colour)
+    self.score_text_rect = self.score_text.get_rect()
+    self.score_text_rect.center = pygame.Rect(40, 770, 80, 40).center
+
+    self.start_text = self.font.render("Press space to start!", True, self.text_colour)
+    self.start_text_rect = self.start_text.get_rect()
+    self.start_text_rect.center = pygame.Rect(0, 400, 550, 50).center
+
+    self.score = 0
+
+  def update_highscore(self):
+    if self.score > self.highscore:
+      highscore = {"highscore": self.score}
+      with open("highscore.json", "w") as file:
```

Comment:
What made you decide to use a json file with a single keyed value to store a single integer?